src/main/java/io/jenkins/plugins/pipelinegraphview/utils/AbstractPipelineViewAction.java

```diff
@@ -62,13 +47,11 @@ public HttpResponse getTree() throws JsonProcessingException {
 
   @Override
   public String getIconFileName() {
-    String iconClassName = getIconClassName();
-    Icon icon = IconSet.icons.getIconByClassSpec(iconClassName + " icon-md");
-    return "/plugin/" + icon.getUrl();
+    return null;
   }
 
   @Override
   public String getIconClassName() {
-    return "icon-pipeline-graph";
+    return "symbol-pipeline plugin-ionicons-api";
```

Comment:
I'm not sold on this one, others look great.

Reply:
What would you prefer?

Reply:
Where is it even coming from? I can't find it in the repo (https://github.com/jenkinsci/ionicons-api-plugin/search?q=filename%3Apipeline, https://github.com/jenkinsci/ionicons-api-plugin/blob/main/docs/additions.md), nor on the upstream website (https://ionic.io/ionicons). If it's a custom Jenkins-specific icon and it doesn't look good, maybe we could still use it but change it (somewhere) to look better?

Reply:
I picked file-tray-stacked-outline